shared/sliceutil/slice.go

```diff
@@ -259,3 +263,23 @@ func IntersectionByteSlices(s ...[][]byte) [][]byte {
 	}
 	return inter
 }
+
+// InsertSort inserts an element into a sorted slice at its index.
```

Comment:
Tests for `InsertSort` and `TruncateItems`?? 😅

Reply:
sure, missed it somehow

Reply:
thanks
Added